File: Client/src/app/services/model/player.ts
import { Team } from './team';
import { Guid } from 'guid-typescript';

export interface Player {
  id: number;
  name: string;
  role: string;
  averageFv: number;
  startValue: number;
  currentValue: number;
  soldValue: number;
  teamFantaId: string;
  teamReal: string;


}

export class PlayerClass {
  constructor(
    public id: number,
    public name: string,
    public role: string,
    public averageFv: number,
    public startValue: number,
    public currentValue: number,
    public soldValue: number,
    public teamFantaId?: Guid,
    public teamReal?: string,
  ) { }
}

export class AddPlayer {
  constructor(
    public id: number,
    public soldValue: number,
    public teamFantaId: string,
  ) { }
}

File: 2.1/fapi/Data/Entities/Player.cs
using System;

namespace fantacalcioApi.Data.Entities
{
    public class Player
    {
        public int Id { get; set; }
        public string Role { get; set; }
        public string Name { get; set; }
        public double AverageFv { get; set; }
        public int StartValue { get; set; }
        public int CurrentValue { get; set; }
        public int SoldValue { get; set; }
        public string TeamFantaId { get; set; }
        public string TeamReal { get; set; }
    }
}